src/components/molecules/AssetStatus.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import Status from '../atoms/Status'
+import Tooltip from '../atoms/Tooltip'
+import styles from './AssetStatus.module.css'
+
+export default function AssetStatus({
+  isDisable
```

Comment:
Please use `isOrderDisabled`, it's more consistent

Reply:
yup, and more so why a boolean and why is it optional? Why is the prop simply setting state when the whole component could be prevented from rendering in the parent component without passing a props?

What is this component supposed to do, what is it supposed to change inside when the prop changes? If nothing changes in component based on prop there seems to be no need for them.